DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -186,6 +186,15 @@ if [[ $DO_TPC_RESIDUAL_EXTRACTION == "1" ]]; then
   export CALIB_CPV_GAIN=0
   export CALIB_ZDC_TDC=0
   export CALIB_FT0_TIMEOFFSET=0
+  if [[ $DO_TPC_RESIDUAL_EXTRACTION == "1" ]]; then
+    export CALIB_TPC_SCDCALIB_SENDTRKDATA=1
+    export CALIB_DIR="./"
```

Comment:
Maybe move this line up, so that its always defined in case `ADD_CALIB==1`?

Reply:
Good idea, done!